lib/std/math.zig

```diff
@@ -646,31 +646,31 @@ test "math.divCeil" {
     comptime testDivCeil();
 }
 fn testDivCeil() void {
-    testing.expect((divCeil(i32, 5, 3) catch unreachable) == 2);
-    testing.expect((divCeil(i32, -5, 3) catch unreachable) == -1);
-    testing.expect((divCeil(i32, 5, -3) catch unreachable) == -1);
-    testing.expect((divCeil(i32, -5, -3) catch unreachable) == 2);
-    testing.expect((divCeil(i32, 0, 5) catch unreachable) == 0);
-    testing.expect((divCeil(u32, 0, 5) catch unreachable) == 0);
+    testing.expectEqual(divCeil(i32, 5, 3) catch unreachable, 2);
```

Comment:
`expectEqual` takes first argument as the expected value (with correct type) and then the value to be tested.